.github/workflows/kind.yml

```diff
@@ -63,3 +85,19 @@ jobs:
           name: e2e-kind-fa.tar.gz
           path: log.tar.gz
           retention-days: 30
+
+  # Runs after all other jobs in the workflow and deletes Theia Docker images uploaded as temporary
+  # artifacts. It uses a third-party, MIT-licensed action (geekyeggo/delete-artifact). While Github
```

Comment:
Not familiar with how temporary artifacts are managed, will there be possible conflicts if multi images are uploaded and deleted?

Reply:
Hi Ziyou, if there are multiple jobs uploading a same image, the later jobs will overwrite what was previously uploaded. I think this won't be a problem for our testing usage?